index.js

```diff
@@ -19,27 +14,35 @@ const npar = spec => {
   }
   return spec
 }
+const mapJson = (value, [key]) => {
+  if (value === 'read') {
+    return [key, 'read-only']
+  } else if (value === 'write') {
+    return [key, 'read-write']
+  } else {
+    return [key, value]
+  }
+}
 
 const cmd = module.exports = {}
 
 cmd.public = (spec, opts) => setAccess(spec, 'public', opts)
 cmd.restricted = (spec, opts) => setAccess(spec, 'restricted', opts)
 function setAccess (spec, access, opts) {
```

Comment:
Should get a default value, `..., opts = {}) {`

Reply:
Added defaults for all the api surface area, with tests